### DISCUSSION TITLE
建议稳定币放弃锚定假想1，以锚定稳定度1做为参考，提供流动性！

### DISCUSSION THREAD

**Discussion by ginxiao** [famland/website#80] [Ideas]
建议修正价格修正参数的１，变成为锚定稳定度1做为参考的YL价格稳定比：
表述为，YL价格稳定比＝周期最小YL价格／周期最大YL价格；
即将绝对值１Ｕ改为相对变化值比稳定度做参考。
则
每秒的当前YL总流速=肥力值rebase的增量/肥力值总量*流动池总资金量*YL价格修正参数/周期时长８６４００秒，式中的YL价格修正参数表述改为：

**n周期YL价格稳定比^m周期次方**
**n<=m，(9<=m)**
**m&n由的参数未来交由社区投票表决，以做为灵度与稳度控制。**

其中，**n周期YL价格稳定比＝n周期最小YL价格／n周期最大YL价格**，
此种情况下推理可得：０＜YL价格稳定比＜=１。

A例：设n=10,m=10，yl价格小于1时,当前价格为0.600
1.如果当前yl价格10个周期的最大值为0.950，当前yl价格10个周期的最小值为0.550，则10周期的稳定比=0.579，说明很不稳定。
则改进后的价格修正参数为0.579^10=0.0042。
原价格修正参数为0.600^9=0.0101，此时前者对于流速只现有参数流速的一半不到，对流速控制更紧。
2.如果当前yl价格10个周期的最大值为0.750，当前yl价格10个周期的最小值为0.550，则10周期的稳定比=0.733，说明基本稳定。
则改进后的价格修正参数为0.733^10=0.0448。
原价格修正参数仍然为0.600^9=0.0101，此时两者对于流速控制相差4倍，说明改进后价格稳定可以带来流量。
3.如果当前yl价格10个周期的最大值为0.555，当前yl价格10个周期的最小值为0.550，则10周期的稳定比=0.990，说明完全稳定，社区生态认可当前价格，当然可以最大流速发行，APY30倍。
则改进后的价格修正参数为0.990^10=0.9044。
原价格修正参数仍然为0.600^9=0.0101，此时两者对于流速控制相差近200倍，说明原机制价格稳定后无法引流，改进后可以引流。

B例：设n=10,m=10，yl价格大于1时，当前价格为6.00时。
1.如果当前yl价格10个周期的最大值为9.50，当前yl价格10个周期的最小值为5.50，则10周期的稳定比=0.579，说明很不稳定。
则改进后的价格修正参数为0.579^10=0.0042。
原价格修正参数为6.00^（1-6.00）=0.0001，此时两者对于流速控制相差40倍，现有对于大于1情况控制很紧。
2.如果当前yl价格10个周期的最大值为7.50，当前yl价格10个周期的最小值为5.50，则10周期的稳定比=0.733，说明基本稳定。
则改进后的价格修正参数为0.733^10=0.0448。
原价格修正参数仍然为6.00^（1-6.00）=0.0001，此时两者对于流速控制相差448倍，相对稳定还是没有收益。
3.如果当前yl价格10个周期的最大值为5.55，当前yl价格10个周期的最小值为5.50，则10周期的稳定比=0.990，说明完全稳定，社区生态认可当前价格，当然可以最大流速发行，APY30倍。
则改进后的价格修正参数为0.990^10=0.9044。
原价格修正参数仍然为6.00^（1-6.00）=0.0001，此时两者对于流速控制相差9000倍，说明原机制价格稳定后无法引流，改进后可以引流。

**Comment by ginxiao** [comment]
通过n,m参数，社区投票还可以调整流速对于价格稳定性跟随的灵敏度与调节阀值，所有权归于社区。而现有的1U锚定所有权归于美联储。
去中心化社区发行的yl，其价格应当适应社区不同阶段的发展。不论大于1或等于都能达到稳定后的最大收益，原方式只等于1才有大收益，这与比特币设立初衷相背，无论此种改进机制是否合适，但1U锚定是无法创造元宇宙的新经济，仍然是旧经济思维的延续。

**Comment by yancanlou** [comment]
上述举例时，n与m的大小关系没按所述设定来，当取n=10，m=20。
这样n代表对稳定度周期波动阀值，而m则代表了对稳定度调节敏度阀值。
